java/compaction/compaction-core/src/main/java/sleeper/compaction/job/status/CompactionJobStartedEvent.java

```diff
@@ -0,0 +1,185 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.compaction.job.status;
+
+import sleeper.compaction.job.CompactionJob;
+
+import java.time.Instant;
+import java.util.Objects;
+
+/**
+ * An event for when a compaction job was started. Used in the compaction job status store.
+ */
+public class CompactionJobStartedEvent {
+
+    private final String jobId;
+    private final String tableId;
+    private final String taskId;
+    private final String jobRunId;
+    private final Instant startTime;
+
+    private CompactionJobStartedEvent(Builder builder) {
+        jobId = Objects.requireNonNull(builder.jobId, "jobId must not be null");
+        tableId = Objects.requireNonNull(builder.tableId, "tableId must not be null");
+        taskId = Objects.requireNonNull(builder.taskId, "taskId must not be null");
+        jobRunId = builder.jobRunId;
+        startTime = Objects.requireNonNull(builder.startTime, "startTime must not be null");
+    }
+
+    /**
+     * Creates a builder for an event that marks the start of a compaction job run.
+     *
+     * @param  job       the compaction job
+     * @param  startTime the start time
+     * @return           an instance of this class
+     */
+    public static Builder compactionJobStarted(CompactionJob job, Instant startTime) {
+        return builder()
+                .job(job)
+                .startTime(startTime);
+    }
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public String getJobId() {
+        return jobId;
+    }
+
+    public String getTableId() {
+        return tableId;
+    }
+
+    public String getTaskId() {
+        return taskId;
+    }
+
+    public String getJobRunId() {
+        return jobRunId;
+    }
+
+    public Instant getStartTime() {
+        return startTime;
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(jobId, tableId, taskId, jobRunId, startTime);
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj) {
+            return true;
+        }
+        if (!(obj instanceof CompactionJobStartedEvent)) {
+            return false;
+        }
+        CompactionJobStartedEvent other = (CompactionJobStartedEvent) obj;
+        return Objects.equals(jobId, other.jobId) && Objects.equals(tableId, other.tableId) && Objects.equals(taskId, other.taskId) && Objects.equals(jobRunId, other.jobRunId)
+                && Objects.equals(startTime, other.startTime);
+    }
+
+    @Override
+    public String toString() {
+        return "CompactionJobStartedEvent{jobId=" + jobId + ", tableId=" + tableId + ", taskId=" + taskId + ", jobRunId=" + jobRunId + ", startTime=" + startTime + "}";
+    }
+
+    /**
+     * Builder for ingest job started event objects.
```

Comment:
The javadoc comments in this builder are still referring to ingest jobs

Reply:
Fixed, thanks